packages/block-editor/src/components/index.js

```diff
@@ -5,6 +5,7 @@ export { default as BlockAlignmentToolbar } from './block-alignment-toolbar';
 export { default as BlockControls } from './block-controls';
 export { default as BlockEdit } from './block-edit';
 export { default as BlockFormatControls } from './block-format-controls';
+export { default as BlockNavigation } from './block-navigation';
 export { default as BlockNavigationDropdown } from './block-navigation/dropdown';
```

Comment:
This dropdown won't be used anywhere in Gutenberg. I feel like we can probably safely remove it as I don't think third party developpers would need this but since it's part of the public API, I kept it for now.